### DISCUSSION TITLE
unable to send images

### DISCUSSION THREAD

**Discussion by bhishmakajaria** [devlikeapro/waha#150] [General]
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/67088711/96ee37bd-bba1-46ec-a886-b8b49d5f7522)

Request you to please help on this as is has stopped send images. also the vcard is still not implemented. can you please share the resolution for this. I have waha plus.

**Comment by allburov** [comment]
Hi! Could you send the full response ir the version field from the response?

**Reply by bhishmakajaria** [reply]
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/67088711/e096b612-a08a-4c1c-aa30-9f315abcde0f)

**Reply by allburov** [reply]
Could you pull  and restart your containers the latest version, it's been fixed there

**Reply by bhishmakajaria** [reply]
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/67088711/6c9e62f6-a5ea-4eaf-8576-cac685695a97)

**Reply by allburov** [reply]
To get the updated version you need to use the last key from post - either on Boosty or Patreon

**Comment by allburov** [comment]
I hope the latest version works here, let me know if it doesn't!
